packages/twenty-front/src/modules/app/effect-components/PageChangeEffect.tsx

```diff
@@ -107,10 +121,16 @@ export const PageChangeEffect = () => {
     if (!previousLocation || previousLocation !== location.pathname) {
       setPreviousLocation(location.pathname);
       executeTasksOnAnyLocationChange();
+      resetIsPageInEditMode();
```

Comment:
Could we put this into `executeTasksOnAnyLocationChange`, that would be a good use case here isn't it ?